Mimick MPDs httpd plugin, while owntone doesn't listen for port 8000 for
this, a proxy in front could map that to stream.mp3 easily.

diff --git a/src/mpd.c b/src/mpd.c
index 10645409..0ac30867 100644
--- a/src/mpd.c
+++ b/src/mpd.c
@@ -3753,6 +3753,14 @@ speaker_enum_cb(struct player_speaker_info *spk, void *arg)
 static int
 mpd_command_outputs(struct evbuffer *evbuf, int argc, char **argv, char **errmsg, struct mpd_client_ctx *ctx)
 {
+  /* emulate MPDs output for http stream, as "httpd" plugin, which some
+   * clients look for to ascertain there is stream playback support */
+  evbuffer_add_printf(evbuf,
+  		      "outputid: 65536\n"          /* USHRT_MAX + 1 */
+  		      "outputname: MP3 stream\n"
+  		      "plugin: httpd\n"
+  		      "outputenabled: 1\n");
+
   player_speaker_enumerate(speaker_enum_cb, evbuf);
 
   return 0;
